pom.xml

```diff
@@ -1430,4 +1430,8 @@
       </plugin>
     </plugins>
   </reporting>
+
+  <prerequisites>
+    <maven>3.2</maven>
+  </prerequisites>
```

Comment:
No, this one was a huge surprise for me, the maven plugin resolution mechanism is quite curious, they assume by default that users have the lowest version possible of maven and resolve the plugin versions from it, in a really conservative manner, it looks for the lowest version of the plugin for those that are not explicit. This can seem redundant but actually it forces the plugin resolution to consider that the user has in the worst case the version defined here.